### DISCUSSION TITLE
Multi Select & Mass Update on all pages ie Devices, Entities, Helpers, Automation, Scenes, Scripts

### DISCUSSION THREAD

**Discussion by cloudbr34k84** [home-assistant/frontend#20257] [Other feature requests]
With the launch of the new organisation, could we please look to enable mass select and mass update. It would be great if we could select the entities and update all of them to a certain area, or labels or category. As we get better and people look to find better ways to organise its quite tedious updating everything one by one.

**Comment by matthiasdebaat** [comment]
Agreed! We have some ideas for this. Nothing in the works yet. Do you've got an idea or example what we could do?

**Reply by cloudbr34k84** [reply]
Well you could do how Sonarr does it
![image](https://github.com/home-assistant/frontend/assets/58960644/7bae73c6-2bfe-4899-806c-9cc9c33ba6ef)

Or you could do something like my Finance App Pocketsmith
![image](https://github.com/home-assistant/frontend/assets/58960644/4ae97e6d-036e-4a1a-84c3-f3062159a854)
![image](https://github.com/home-assistant/frontend/assets/58960644/735a26b0-5217-4dcf-9999-16d7e7a312f3)
